subsys/event_manager/event_manager_shell.c

```diff
@@ -50,33 +47,31 @@ static int show_listeners(const struct shell *shell, size_t argc,
 static int show_subscribers(const struct shell *shell, size_t argc,
 		char **argv)
 {
-	shell_fprintf(shell, SHELL_NORMAL, "Registered Subscribers:\n");
+	shell_fprintf(shell, SHELL_NORMAL,
+			"Registered Subscribers, execution order from top to bottom:\n");
```

Comment:
I don't think `, execution order from top to bottom` is needed

Reply:
It somehow subsidies printing priority. If you do not like it I can remove it.

Reply:
I removed it in the review commit